adafruit_framebuf.py

```diff
@@ -282,8 +282,7 @@ def image(self, img):
         # Clear buffer
         for i in range(len(self.buf)):
             self.buf[i] = 0
-        # Iterate through the memory pages
-        index = 0
+        # Iterate through the pixels
         for x in range(self.width):       # yes this double loop is slow,
```

Comment:
Possibly change in the nested loop, `imwidth` should be valid as they're checked above
`self.width` to `imwidth`

Reply:
yes they must be identical